core/Pos/Core/Arbitrary.hs

```diff
@@ -71,25 +73,30 @@ deriving instance Random Types.EpochIndex
 
 instance Arbitrary Types.EpochIndex where
     arbitrary = choose (0, maxReasonableEpoch)
+    shrink = genericShrink
 
 instance Arbitrary Types.LocalSlotIndex where
     arbitrary =
         leftToPanic "arbitrary@LocalSlotIndex: " . Types.mkLocalSlotIndex <$>
         choose (Types.getSlotIndex minBound, Types.getSlotIndex maxBound)
+    shrink = genericShrink
 
+-- NOTE: Deriving an 'Arbitrary' instance for wrappers over integral types with 'derive
```

Comment:
First, this instance uses `genericArbitrary`, not `derive makeArbitrary`. Second, you are repeating yourself. To have the same commentary in multiple places there's a good practice popularized by GHC devs to leave so called "NOTE"s — you can read about them here - https://ghc.haskell.org/trac/ghc/wiki/Commentary/CodingStyle#Commentsinthesourcecode

I also recall that @pva701  used them somewhere in our codebase (grep the sources for `~~~~`).